script.js

```diff
@@ -0,0 +1,442 @@
+const myobj={};
+
+function appendToInput(value) {
+    const textInput = document.getElementById('textInput');
+
+    const s = Array.from(textInput.value);
+
+    if(value !== "+" && value !== "-" && value !== "*" && value !== "/" && value !== "^" &&
```

Comment:
Create a Array constant for all the operators like +, -, * etc and use !operators.includes(value)